frontend/packages/console-shared/src/components/status/__tests__/status.test.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { Status } from '../status';
+import StatusFixtures from '../fixtures/status.fixture';
+
+const testStatus = () => <Status {...StatusFixtures[0].props} />;
+
+describe('<Status />', () => {
+  it('renders correctly', () => {
+    const component = shallow(testStatus());
+    expect(component).toMatchSnapshot();
```

Comment:
The snapshot seems to have a lot of noise.
I also see little value in this test case.

Reply:
I think this relates to the common practice where each React component has (at least) a simple snapshot test to ensure the rendered markup doesn't change unexpectedly.

As for the messy snapshot file, there are things which don't make sense to serialize (like symbols), meaning we should update Jest & Enzyme & related dependency versions and ensure the generated snapshots are meaningful.

Console doesn't currently have any snapshot tests (`git grep 'toMatchSnapshot'` returns nothing) so we should revisit the Enzyme (to-JSON) serializer stuff here. This also relates to #1648 which I'm revisiting at the moment.